### PR TITLE
use when@prod for sentry

### PR DESCRIPTION
**Pull request by robinlehrmann**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: sentry/sentry-symfony/3.0/config/packages/sentry.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        options:
+            excluded_exceptions:
+                - Symfony\Component\HttpKernel\Exception\NotFoundHttpException
+                - Symfony\Component\Security\Core\Exception\AccessDeniedException
+
+        # If you are using Monolog, you also need these additional configuration and services to log the errors correctly:
+        # https://docs.sentry.io/platforms/php/guides/symfony/#monolog-integration
+        # register_error_listener: false
+
+        # monolog:
```
Comment (nicolas-grekas) [inline sentry/sentry-symfony/3.0/config/packages/sentry.yaml:13]: extra space at start of line

File: sentry/sentry-symfony/3.0/config/packages/sentry.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        #    handlers:
+        #        sentry:
+        #            type: service
+        #            id: Sentry\Monolog\Handler
+
+        # services:
```
Comment (nicolas-grekas) [inline sentry/sentry-symfony/3.0/config/packages/sentry.yaml:19]: extra space

File: sentry/sentry-symfony/3.0/config/packages/sentry.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        #    Sentry\Monolog\Handler:
+        #        arguments:
+        #           $hub: '@Sentry\State\HubInterface'
+        #            $level: !php/const Monolog\Logger::ERROR
```
Comment (nicolas-grekas) [inline sentry/sentry-symfony/3.0/config/packages/sentry.yaml:23]: please ensure that the indentation is always 4 spaces in all examples (with no leading space for the first line of yaml)

**Review by nicolas-grekas — COMMENTED** [review]
please ensure that the indentation is always 4 spaces in all examples (with no leading space for the first line of yaml)

**Comment by robinlehrmann** [comment]
Thanks for the review @nicolas-grekas 
I fixed what you mentioned and hope that everything is correct now. If not, feel free to tell me. :)

**Comment by robinlehrmann** [comment]
> The change on 3.0 looks wrong isn't it? I submitted #1420 with what looks like the correct patch to me. WDYT?

Oh.. you are right 😅 yes, looks good :)

**Pull request closed without merging**